templates/web/src/payload.ts.twig

```diff
@@ -0,0 +1,40 @@
+export class Payload extends Blob {
+    public name: string | null = null;
```

Comment:
In Node we do it private, and have getName method. No idea which is better, lets just be consistent since they are both same language

Reply:
It's idiomatic to just have a public field as an accessor, imo

having a `getName` linked to a private field is very enterprise java